tests/common/fixtures/services.py

```diff
@@ -2,21 +2,37 @@
 
 import pytest
 
+from h.services.annotation_delete import AnnotationDeleteService
 from h.services.annotation_moderation import AnnotationModerationService
+from h.services.delete_group import DeleteGroupService
+from h.services.group import GroupService
+from h.services.group_create import GroupCreateService
+from h.services.group_members import GroupMembersService
+from h.services.group_update import GroupUpdateService
 from h.services.groupfinder import GroupfinderService
 from h.services.links import LinksService
+from h.services.list_organizations import ListOrganizationsService
 from h.services.nipsa import NipsaService
+from h.services.organization import OrganizationService
 from h.services.search_index import SearchIndexService
 from h.services.search_index._queue import Queue
 
 __all__ = (
     "mock_service",
-    "search_index",
-    "nipsa_service",
-    "user_service",
+    "annotation_delete_service",
+    "delete_group_service",
     "links_service",
+    "list_organizations_service",
     "groupfinder_service",
+    "group_create_service",
+    "group_members_service",
+    "group_service",
+    "group_update_service",
+    "nipsa_service",
     "moderation_service",
+    "organization_service",
+    "search_index",
+    "user_service",
```

Comment:
This is mostly the addition of new services, but I also sorted the alphabetically, as we're getting a comically large number.

I really think we could do with merging the group services. Especially seeing how it forces us to look up and mock five different ones in the code and tests.